sdks/go/pkg/beam/core/runtime/harness/init/init.go

```diff
@@ -51,7 +51,7 @@ var (
 	controlEndpoint = flag.String("control_endpoint", "", "Local control gRPC endpoint (required in worker mode).")
 	//lint:ignore U1000 semiPersistDir flag is passed in through the boot container, will need to be removed later
 	semiPersistDir = flag.String("semi_persist_dir", "/tmp", "Local semi-persistent directory (optional in worker mode).")
-	options        = flag.String("options", "", "JSON-encoded pipeline options (required in worker mode).")
+	options        = flag.String("options", "", "JSON-encoded pipeline options (required in worker mode). (deprecated)")
```

Comment:
removed?

Reply:
It's useful to not fully break everything in one version if something is going wrong. Leaving the flag in allows older containers to be used as a backup.

Reply:
Got it. worth calling this change out in our CHANGES.md?

Reply:
Not really. It's not user facing.

If we also do Python it's worth mentioning since the work is then complete.